GitCommands/Config/ConfigFile.cs

```diff
@@ -454,6 +454,7 @@ private ParsePart ReadValue(char c)
             {
                 if (_escapedValue)
                 {
+                    // TODO: case 'b' is valid, according to https://git-scm.com/docs/git-config#_syntax
```

Comment:
Should it be handled in this PR? Else a new issue should be opened.

Reply:
@mstv 
Seems like an unrelated change to me, maybe an issue should be raised.

Reply:
Agree, please remove from this PR